client/reader/stream/index.jsx

```diff
@@ -437,15 +465,78 @@ class ReaderStream extends Component {
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 				/>
 			);
+			let sidebarContent = <ReaderSidebarFollowedSites path={ path } />;
+			if ( 'a8c' === streamType || 'p2' === streamType ) {
+				sidebarContent = (
+					<ReaderSidebarOrganizations
+						organizations={ this.props.organizations }
+						path={ path }
+						translate={ translate }
+					/>
+				);
+			}
+
+			// Only show right sidebar on select screens
+			const excludesSidebar = [
```

Comment:
Looks like something that doesn't need to be redeclared on every render and can be moved outside of the component as a separate constant.